server/static/script.js

```diff
@@ -152,5 +154,5 @@ function showError() {
     // Reset back to the home screen after one second.
     setTimeout(() => {
         showHome();
-    }, 1000);
+    }, 3000);
```

Comment:
What's the reason for upping the failure screen? In my mind, folks will want to get back to the default QR code as quickly as possible if they choose the wrong emoji on accident.

Reply:
It seemed too short when I experienced how short 3 secs on success was